lib/fog/compute/terremark/models/server.rb

```diff
@@ -171,9 +171,9 @@ def save
               server_name=name,
               vapp_template=image,
               options={
-                  'ssh_key_fingerprint' => sshkeyFingerPrint,
-                  'cpus' => vcpus,
-                  'memory' => memory
+                  "ssh_key_fingerprint" => sshkeyFingerPrint,
```

Comment:
Use 2 spaces for indentation in a hash, relative to the start of the line where the left curly brace is.